scripts/run.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/sh
+
+# ======================================= FUNCTIONS START =======================================
+
+# ======================================= BE QUIET PUSHD/POPD ===================================
+pushd () {
+    command pushd "$@" > /dev/null
+}
+
+popd () {
+    command popd "$@" > /dev/null
+}
+
+function print_usage() {
+cat <<EOF
+
+USAGE: run.sh -p <PLATFORM> -b <BINDER> [-s -t -c]
+  The default mode will setup, run tests and clean up, you can control which stage you want to
+  have executed by toggling the flags (-s, -t, -c)
+
+Flags:
+
+[*] -p  | --platform - define the target platform to run
+    -b  | --binder - define the binder (i.e. RABBIT, KAFKA) defaults to RABBIT
+    -s  | --skipSetup - skip setup phase
+    -t  | --skipTests - skip test phase
+    -c  | --skipCleanup - skip the clean up phase
+
+[*] = Required arguments
+
+EOF
+}
+
+function load_file() {
+filename=$1
+
+while IFS='=' read -r var value; do
+  if [ -z ${!var} ]; then
+    export $var=$value
+  fi
+done < "$filename"
+}
+
+function test_port() {
+  nc -z -w1 ${1} $2
+}
+
+function netcat_port() {
+    local READY_FOR_TESTS=1
+    for i in $( seq 1 "${RETRIES}" ); do
+        nc -z -w1 ${1} $2 && READY_FOR_TESTS=0 && break
+        echo "Fail #$i/${RETRIES}... will try again in [${WAIT_TIME}] seconds" >&2
+        sleep "${WAIT_TIME}"
+    done
+    return ${READY_FOR_TESTS}
+}
+
+function download(){
+  if [ ! -f "$1/scdf-server.jar" ]; then
+    echo "Downloading latest release from $SPRING_CLOUD_DATAFLOW_SERVER_DOWNLOAD_URL"
+    wget $SPRING_CLOUD_DATAFLOW_SERVER_DOWNLOAD_URL --progress=bar -O $1/scdf-server.jar
+  fi
+}
+
+function run_scripts()
+{
+  pushd $1
+   . $2
+  popd
+}
+
+function setup() {
+  pushd $PLATFORM
+    run_scripts "init" "setenv.sh"
+    run_scripts "mysql" "create.sh"
+    pushd "binder"
+      run_scripts $BINDER "create.sh"
+    popd
+    run_scripts "redis" "create.sh"
+    run_scripts "server" "create.sh"
```

Comment:
May want to add a health check to make sure Dataflow is up and running prior to test execution.

i.e. https://github.com/spring-cloud/spring-cloud-dataflow-acceptance-tests/blob/master/startLocalAcceptance.sh#L330